tests/cases/go_server/empty_schema/swagger.yaml

```diff
@@ -0,0 +1,93 @@
+swagger: '2.0'
```

Comment:
Is there a reason why you define many types in this test cases instead of just defining a single empty definition (that would be `meta`, correct?)? 

I wrote a suggestion (see at the bottom of this file) that is a bit more succinct and tests only for empty schemas. Please double-check that it was what you intended, I might have missed an important detail.